cabal-install/src/Distribution/Client/ProjectBuilding/UnpackedPackage.hs

```diff
@@ -678,7 +679,32 @@ buildAndInstallUnpackedPackage
           runConfigure
         PBBuildPhase{runBuild} -> do
           noticeProgress ProgressBuilding
-          runBuild
+          hookDir <- (</> ".cabal/hooks") <$> getEnv "HOME"
```

Comment:
I think this isn't quite what we want. Cabal has its own logic (e.g. XDG dir, honoring config values, etc.). There must be a set of functions providing those locations.

Reply:
Replaced this with:
```
          cabalDir <- dropFileName <$> defaultConfigFile
          let hookDir = cabalDir </> "hooks"
```

Reply:
I suggest `getDefaultDir XdgConfig "hooks"`.

Reply:
This comment has been made redundant since the hooks are now project local.